src/main/java/org/example/ArrayFunctions.java

```diff
@@ -0,0 +1,114 @@
+package org.example;
+
+import java.util.Random;
+
+public class ArrayFunctions {
+
+    //Creation of random array with random length (1-10) and random ints with values (-100 -> +100)
+    public static int[] creationOfRandomArray() {
+        Random random = new Random();
+        int max_length_size = 10;
+        int max_int_size = 200;
+        int length = random.nextInt(max_length_size) + 1;
+        int[] array = new int[length];
+        for (int i = 0; i < array.length; i++) {
+            array[i] = random.nextInt(max_int_size) - 100;
+        }
+        return array;
+    }
+
+    //Creation of array with numbers set by user inputs
+    public static int[] creationOfArraySetByUser(String input) {
+        String[] arrayOfStringNumbers = input.split(",");  //Splits whole String into array of StringNumbers
+        int[] array = new int[arrayOfStringNumbers.length];
+
+        for (int i = 0; i < arrayOfStringNumbers.length; i++) {
+            try {
+                array[i] = Integer.parseInt(arrayOfStringNumbers[i].trim());  //Remakes every StringNumber to Integer
+            } catch (NumberFormatException e) {
+                System.out.println("Invalid input: '" + arrayOfStringNumbers[i] + "' is not a valid integer.");
+                array = new int[0];
+            }
+        }
+        return array;
+    }
+
+    //Printing of chosen array
+    public static void arrayPrinting(int[] array) {
+        for (int i = 0; i < array.length; i++) {
+            System.out.print(array[i] + " ");
+        }
+        System.out.println();
+    }
+    //Adding new number to existing array
+    public static int[] addingNumberToArray(int[] array, int number) {
+        int[] newArray = new int[array.length + 1];
+        for (int i = 0; i < array.length; i++) {
+            newArray[i] = array[i];
+        }
+        newArray[array.length] = number;
+        return newArray;
+    }
+
+    //Looking for the max number from given array
+    public static int findingMaxNumber(int[] array) {
+        int maxNumber = array[0];
```

Comment:
Tady ti program v určité chvíli vyhodí: java.lang.ArrayIndexOutOfBoundsException: Index 0 out of bounds for length 0